frontend/src/modules/admin/modules/projects/components/form/lf-project-form.vue

```diff
@@ -184,6 +211,7 @@ const isEditForm = computed(() => !!props.id);
 const fillForm = (record?: Project) => {
   if (record) {
     Object.assign(form, record);
+    form.type = record.isLF ? 'LF' : 'nonLF';
```

Comment:
If you could put LF and nonLF to enum would be amazing. You can have this enum just inside this componetnt if you use it just here